backend/tss_api/src/api/keplr_auth/index.ts

```diff
@@ -21,6 +21,7 @@ export function generateUserToken(
     const payload: UserTokenPayload = {
       email: args.email,
       wallet_id: args.wallet_id,
+      wallet_id_ed25519: args.wallet_id_ed25519,
```

Comment:
A separate wallet_id_ed25519 was required because each curve type is stored as a distinct wallet record in the database. Do you have a better suggestion?